OWML.ModHelper/ModConfig.cs

```diff
@@ -4,6 +4,7 @@
 using Newtonsoft.Json;
 using Newtonsoft.Json.Linq;
 using OWML.Common;
+using UnityEngine.EventSystems;
```

Comment:
are you using this, and if yes: for what?

Reply:
probably got added automatically, will remove that